src/arch/x86_64/encoder.zig

```diff
@@ -117,7 +117,7 @@ pub const Instruction = struct {
 
     pub fn new(mnemonic: Mnemonic, args: Init) !Instruction {
         const encoding = (try Encoding.findByMnemonic(mnemonic, args)) orelse {
-            log.debug("no encoding found for: {s} {s} {s} {s} {s} {s}", .{
```

Comment:
If you want to make this `log.err` we really should hook up to the compiler's error mechanism instead.

Reply:
Probably, but currently it is really annoying to find the one `no encoding found` among the 102246 `encoding found` debug messages, and this makes development much easier in the mean time.

Reply:
Makes sense! FWIW I will be rolling out the error mechanism in the MachO linker so can keep you posted how that goes.